benchmarks/routines/gemm.py

```
@@ -1233,3 +1246,412 @@ def run_backend(
+                    reference_output.reshape(-1),
+                    tested_outputs[i].reshape(-1),
+                    dim=0,
+                )
+                if cos_sim < 0.99:
+                    print(
+                        f"[ERROR] Output tensor mismatch from backend {tested_backends[i]} with cos_sim={cos_sim}"
+                    )
+                    if not args.allow_output_mismatch:
+                        raise AssertionError(
+                            f"[ERROR] Backend {tested_backends[i]} output mismatch with cos_sim={cos_sim}"
+                        )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)